src/components/ArmyBuildPane.jsx

```diff
@@ -13,12 +16,34 @@ const Container = styled.div`
     margin: 30px
 `;
 
-const ArmyBuildPane = () => (
-  <Container>
-    <UsageBar />
-    <SelectionPane />
-    <ArmyList />
-  </Container>
-);
+const ArmyBuildPane = () => {
+  const [unitList, setUnitList] = useState({});
+  const [listIndex, setListIndex] = useState(1);
+
+  const addUnit = unit => {
+    setUnitList({
+      ...unitList,
+      [listIndex]: unit
+    });
+    setListIndex(listIndex + 1);
+  };
+  const removeUnit = index => {
```

Comment:
Something is wrong here. Removing is clearing out everything and messing up calculate usage.

Reply:
`Removing is clearing out everything` what does this part mean? Removing what?

Reply:
Basically right functionality is: Click list item in left pane, adds one to right pane(and updates totals in top bar). Clicking an item in right pane should remove that item and update totals but is currently removing all items and breaking some totals :P

Reply:
Turned out trying to take a short for now caused this problem, replaced placeholder with new comp and all is well.